src/gui/gui.config

```diff
@@ -40,6 +40,20 @@
   <camera_pose>6 0 6 0 0.5 3.14</camera_pose>
 </plugin>
 
+<plugin filename="GzSceneManager" name="Scene Manager">
+  <ignition-gui>
+    <anchors target="3D View">
+      <line own="right" target="right"/>
+      <line own="top" target="top"/>
+    </anchors>
+    <property key="resizable" type="bool">false</property>
+    <property key="width" type="double">5</property>
+    <property key="height" type="double">5</property>
+    <property key="state" type="string">floating</property>
+    <property key="showTitleBar" type="bool">false</property>
+  </ignition-gui>
+</plugin>
```

Comment:
When these two plugins are merged in ign-gui we should add them here

 - https://github.com/ignitionrobotics/ign-gui/pull/231
 - https://github.com/ignitionrobotics/ign-gui/pull/226

Reply:
I removed the changes to `gui.config` for now and added an example world that uses the new plugins. This way, we can incrementally add functionality to the scene and only make the switch to it once it's ready.

Added these plugins to the new `minimal_scene.sdf` in fbceade